tests/auth_tests/test_apps.py

```diff
@@ -0,0 +1,58 @@
+import os
+import subprocess
+import sys
+import tempfile
+import unittest
+
+from django.db import ConnectionHandler
+
+
+class AppConfigTestCase(unittest.TestCase):
+    def test_no_migrations(self):
+        pythonpath = [os.path.join(os.path.dirname(__file__), 'nomigrations')]
+        try:
+            env_pythonpath = os.environ['PYTHONPATH']
+        except KeyError:
+            pass
+        else:
+            pythonpath.append(env_pythonpath)
+
+        env = os.environ.copy()
+        env['PYTHONPATH'] = os.pathsep.join(pythonpath)
+
+        # Avoid NamedTemporaryFile as Windows can't reopen the file in the
```

Comment:
I'm not sure if this type of comment is useful or not. Is it likely that someone would try to refactor this?